changelog.txt

```diff
@@ -1,123 +1,263 @@
-		pos-file in python 3.5
-  - Fix bug, which occured when constructing frames from
-	  raw frames using box instances instead of a box matrix.
+[0.3.0] -- 2016-07-06
+---------------------
 
-0.3.3:
+Added
++++++
+  - Provide a highly optimized cythonized ``DCDFileReader``.
+  - Allow trajectory data acess via coherent numpy arrays.
+  - Make snapshot creation and copying HOOMD-blue 2.0 compatible.
 
-	- Fix bug related to 2-dimensional systems and a box
-	  z-dimensions not equal to 1.
+Changed
++++++++
 
-0.3.2:
+  - Update the GSD module.
+  - Improve the ``Box`` class documentation.
+  - Overall improvement of the documentation.
 
-	- Add trajectory.N, trajectory.type and trajectory.type_ids as
-		an alternative mode to access frame length and type information.
-	- Fix bug in GSDHOOMDFileReader when not providing template frame.
+Fixed
++++++
+  - Fix and optimize the pure-python ``DCDFileReader``.
 
-0.3.1:
+Version 0.2
+===========
 
-	- Update the GSD hoomd module.
+[0.2.1] -- 2016-07-10
+---------------------
 
-0.3.0:
+Fixed
++++++
+  - Fix an issue with injavis pos-files causing parser errors.
 
-	- Allow trajectory data acess via coherent numpy arrays.
-	- Provide a highly optimized cythonized DCDFileReader.
-	- Fix and optimize the pure-python DCDFileReader.
-	- Make snapshot creation and copying HOOMD-blue 2.0 compatible.
-	- Update the GSD module.
-	- Improve the Box class documentation.
-	- Overall improvement of the documentation.
+[0.2.0] -- 2016-04-28
+---------------------
 
-0.2.1:
+Fixed
++++++
+  - Fix HOOMD-blue snapshot type issue.
 
-	- Fix an issue with injavis pos-files causing parser errors.
+Version 0.1
+===========
 
-0.2.0:
+[0.1.9] -- 2016-04-09
+---------------------
 
-	- Fix HOOMD-blue snapshot type issue.
+Added
++++++
+  - Add ``GSDHoomdFileReader``.
 
-0.1.9
+Fixed
++++++
+  - Fix type issue in ``HoomdBlueXMLFileReader``.
 
-	- Add GSDHoomdFileReader
-	- Fix type issue in HoomdBlueXMLFileReader
+[0.1.8] -- 2016-04-04
+---------------------
 
-0.1.8
+Added
++++++
+  - Add ``HoomdBlueXMLFileReader``.
+  - Add ``DCDFileReader``.
+  - Add ``CifFileWriter``.
+  - Add ``GetarFileReader``.
 
-	- Fix type issue in DCDFileReader
-	- Fix num issue in HoomdBlueXMLFileReader
-	- Tolerate pos-files without final eof declaration
+Fixed
++++++
+  - Fix type issue in DCD.
 
-0.1.7
 
-	- Add HoomdBlueXMLFileReader
-	- Add DCDFileReader
-	- Add CifFileWriter
-	- Add GetarFileReader
+[0.1.6] -- 2016-01-28
+---------------------
 
-0.1.6
+Changed
++++++++
+  - Extend FileFormat API to increase file-like compatibility.
 
-	- Fix: box_matrix calculation.
-	- Extend FileFormat API to increase file-like compatibility.
+Fixed
++++++
+  - Fixed ``box_matrix`` calculation.
 
-0.1.5
+[0.1.5] -- 2016-01-11
+---------------------
 
-	- Frames only loaded into memory on demand.
-	- Improved trajectory iteration logic.
+Changed
++++++++
+  - Frames only loaded into memory on demand.
+  - Improved trajectory iteration logic.
```

Comment:
Maybe we can add a line here stating that there are no changelog entries prior to this release. 

See freud example: https://github.com/glotzerlab/freud/blob/master/ChangeLog.md#no-change-logs-prior-to-v040

Reply:
5547ab8